Makefile

```diff
@@ -1,10 +1,10 @@
-include dev.env
+include .env
```

Comment:
Will this work, if the file is not yet there? E.g. will one be able to run `git clone ...; cd dbt-athena; make .env`

Reply:
Yes, it works. There is indeed this error :
```
Makefile:1: .env: No such file or directory
```

But the file is correctly created.

Reply:
But the tests cannot run, as the file wasn't read in when the makefile started and anyway are not adjusted.

Reply:
The tests will run but will fail because the dbt profile is wrong, which is the same as : not having an .env file or having an .env file but with env variables not pointing to real AWS settings.